src/package/midlrt/readme.md

```diff
@@ -0,0 +1,26 @@
+## The Windows Runtime (WinRT) API Generation tool
```

Comment:
update title

Reply:
to what exactly? i thought this was a fairly good description of what MIDL does?

Reply:
it's API generation, not projection generation

Reply:
oh n.m. - thought this had been copied from the ABI tool